ui/src/containers/AppsTableContainer.js

```diff
@@ -30,7 +30,7 @@ export class AppsTableContainer extends React.Component {
   getTable () {
     return (
       <div className="apps-table">
-        <AppsTable columns={this.props.columns} rows={this.props.apps.rows} sortBy={this.sortBy} onSort= {this.onSort} onRowClick={this.onRowClick}/>
+        <AppsTable columns={this.props.columns} rows={this.props.apps.rows} sortBy={this.props.sortBy} onSort= {this.onSort} onRowClick={this.onRowClick}/>
```

Comment:
Nice spot on that one. Strange that it didn't break.

Reply:
Yeah, was surprised too. I think React was catching it in the background, but best not to depend on that :)